docs/operators/xmlParserOperator.md

```diff
@@ -0,0 +1,83 @@
+Xml Parser
+=============
+
+## Operator Objective
+Purpose of XmlParser operator is to parse XML records and construct a Plain Old Java Object ("POJO") out of it. The operator also emits each record as DOM Document if the relevant output port is connected. User can also provide a XSD (XML Schema Definition) to validate incoming XML records. Valid records will be emitted as POJO / DOM Document while invalid ones are emitted on error port with the error message if the error port is connected.
```

Comment:
Suggest rewriting first sentence:
The `XmlParser` operator parses XML records and constructs POJOs ("Plain Old Java Objects") from them.
as DOM => as a `DOM`
emitted as POJO / DOM Document => emitted as POJOs / DOM Documents
emitted on error port with the => emitted on the error port with an